baggers.js

```diff
@@ -1,6 +1,47 @@
 // List of baggers...
 var data = {
   speakers: [
+    {
+      since: '2020-26-02',
+      name: 'Thomas Tourlourat',
+      bio:
+        'Lead-dev chez Copines de Voyage, ex @Konbini, ex @Freelance',
+      picture:
+        'https://pbs.twimg.com/profile_images/714783052052971520/WaNkikC9_400x400.jpg',
+      websites: [
+        {
+          name: 'Github',
+          url: 'https://github.com/armetiz',
+        },
+        {
+          name: 'Twitter',
+          url: 'https://twitter.com/armetiz',
+        },
+        {
+          name: 'Malt',
```

Comment:
`Malt` is not one of the supported websites. Check here: https://github.com/brownbaglunch/bblfr_data/#websites